### PR TITLE
Updated README with content

### PR DESCRIPTION
**Pull request by williamsantosa**
Finished most of content. Some parts will require @dinoplane to complete.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,35 +130,32 @@ There are plenty of Professors at UCSC, all of which are notable in their own fi
...
-1. Learn a language (Rec: Python & C).
-2. Learn a second language. Compare and contrast the two, understand what it fully going on underneath everything (like JavaScript's `in` vs. `of` when iterating over an object or how values are being stored in certain programming languages).
-3. Read over some proofs.
-4. Learn common data structures and algorithms (DSAs).
+Having come into University with minimal Computer Science experience, there are a few I would recommend people to do.
 
-## Planning for Graduate School
-
-- Graduate school requirements
+1. Take AP Computer Science (if possible) in High School.
```
Comment (dinoplane) [inline README.md:166]: Community college too... also saves money

File: README.md [hunk 2]
```diff
@@ -130,35 +130,32 @@ There are plenty of Professors at UCSC, all of which are notable in their own fi
...
-- Research is important for graduate school, although not required for a masters degree
-- Talk to Professors (look below)
+There are a few things you should prepare if you are planning on going to graduate school. Namely, talking to professors, having a good GPA, and getting experience in research.
 
 ### Talking to Professors
 
-- You NEED 3 recommendation letters for most graduate schools
-- Make sure to talk with some professors and do well in their classes
-- If you get research experience, even better, can ask them for rec letter later
+Most graduate schools require 3 recommendation letters from people you have worked with. Consequently, you should try to talk or even work with some of the professors at UCSC. A good way to work with them is to be active and get good grades in their class, then ask if they have any projects they need help on. Alternatively, you can also tutor for them and that'll also increase your chances of getting a recommendation letter from them. If there are no opportunities to do so, it also doesn't hurt to ask them for a recommendation letter if you did good in their class.
```
Comment (dinoplane) [inline README.md]: Office hours office hours office hours. also some profs are very busy, but they dont mind you shooting an email or 2

File: README.md [hunk 3]
```diff
@@ -192,11 +188,12 @@ Lastly, research experience is also important for graduate school, although not
 
 #### Study Locations
 
-- Introduce study locations
-- Science & Engineering (with friends)
-- McHenry (with friends)
-- Game Lab (if you have access)
-- Talk about how you can reserve a spot, add links
+There are a few study locations:
```
Comment (dinoplane) [inline README.md:221]: DARC 308 (when class is not in session)

**Review by dinoplane — APPROVED** [review]
ok.

File: README.md (hunk 2 above)
Comment (williamsantosa) [reply README.md]: added

**Pull request merged**